docs/source/dev/styleguide.rst

```diff
@@ -31,22 +31,53 @@ To format all files in your working copy, you can run this command in bash from
      -o -iname "*.loader" -o -iname "*.param" -o -iname "*.unitless" \
      | xargs clang-format-11 -i
 
-Format Changes Using Git
-^^^^^^^^^^^^^^^^^^^^^^^^
+Format Only Changes, Using Git
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Instead of using the bash command above you can use *Git* together with *ClangFormat* to format your patched code only.
+
+    *ClangFormat* is an external tool for code formating that can be called by *Git* on changed files only and
+    is included in the clang library.
+
 Before applying this command, you must extend your local git configuration **once** with all file endings used in *PIConGPU*:
 
 .. code-block:: bash
 
    git config --local clangFormat.extensions def,h,cpp,cu,hpp,tpp,kernel,loader,param,unitless
 
+After installing, or on a cluster loading the module, clang and clangFormat can be called by git on all **staged files** using the command:
+
+.. code-block:: bash
+
+   git clangFormat
+
 .. warning::
 
-    The binary for *ClangFormat* is on some operating systems called `clang-format`.
-    If so please check that `clang-format --version` returns version `11.X.X`.
+    The binary for *ClangFormat* is instead called `clang-format` on some operating systems.
+    If *clangFormat* is not recognized try *clang-format* instead, in addition please check that `clang-format --version` returns version `11.X.X` in this case.
+
+The Typical workflow using git clangFormat is the following,
+
+1. make your patch
+2. stage the changed files in git
+.. code-block:: bash
```

Comment:
a block requires an empty line before the command.

You can preview your PR if you scroll down to the tests, one test is building the documentation.
For the current commit the documentation can be found: https://picongpu--3522.org.readthedocs.build/en/3522/dev/styleguide.html?highlight=clang%20format#format-only-changes-using-git

Reply:
fixed, and thanks for the pointer